src/main/java/by/bookify/book/exception/BookExceptionHandler.java

```diff
@@ -14,4 +15,9 @@ public class BookExceptionHandler {
     public ResponseEntity<ExceptionResponse> handleBookNotFound(BookNotFoundException e) {
         return ResponseEntity.status(NOT_FOUND).body(new ExceptionResponse(e.getMessage()));
     }
+
+    @ExceptionHandler(NotEnoughBooksInStorageException.class)
+    public ResponseEntity<ExceptionResponse> handleNotEnoughBooksInStorage(NotEnoughBooksInStorageException e) {
+        return ResponseEntity.status(INTERNAL_SERVER_ERROR).body(new ExceptionResponse(e.getMessage()));
```

Comment:
INTERNAL_SERVER_ERROR indicates that the problem occured on the server side. Even though it's not a server side error because client knows the storage number left (availability to order in other words). I would indicate this as a client error because he quering something that doesn't exist. 400 Bad Request is appropriate I think. May be you can offer something better. I hope you understand me